Traceback (most recent call last):
  File "C:/Users/Admin/AppData/Local/Programs/Python/Python37/проги1/report_maker/main.py", line 62, in main
    app.start()
  File "C:/Users/Admin/AppData/Local/Programs/Python/Python37/проги1/report_maker/main.py", line 42, in start
    self.init_ui()
  File "C:/Users/Admin/AppData/Local/Programs/Python/Python37/проги1/report_maker/main.py", line 37, in init_ui
    height=2.5)
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python37\lib\tkinter\__init__.py", line 2366, in __init__
    Widget.__init__(self, master, 'button', cnf, kw)
  File "C:\Users\Admin\AppData\Local\Programs\Python\Python37\lib\tkinter\__init__.py", line 2296, in __init__
    (widgetName, self._w) + extra + self._options(cnf))
_tkinter.TclError: expected integer but got "2.5"
